src/main/java/com/jcabi/jdbc/JdbcSession.java

```diff
@@ -309,6 +310,25 @@ public void commit() throws SQLException {
     }
 
     /**
+     * Call an SQL stored procedure.
+     *
+     * <p>JDBC connection is opened and, optionally, closed by this method.
```

Comment:
@amihaiemil right, so can you put the above information to Javadoc? It's more accurate than saying `connection is opened and, optionally, closed`